source/mapping-rules/fields/data-type-conversion-guide.txt

```diff
@@ -0,0 +1,229 @@
+.. _rm-data-type-conversion-guide:
+
+==========================
+Data Type Conversion Guide
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This page outlines examples that show the result of converting the 
+source value and data-type to a specific BSON data-type using customized 
+fields.
+
+Behavior
+~~~~~~~~
+
+Errors during a sync job result in the field being ``BsonNull``. 
```

Comment:
(wording)
This first sentence reads a bit awkward to me. What about a slight tweak. Something like:
"Sync job errors result in ``BsonNull`` values for the impacted fields."